src/app/teams/teams.component.html

```diff
@@ -1,14 +1,16 @@
 <mat-spinner *ngIf="!isLoaded()" color="accent"></mat-spinner>
 <div *ngIf="isLoaded()" class="grid-container">
-<mat-card>
-<mat-card-header>
-  <mat-card-title>Teams</mat-card-title>
-</mat-card-header>
-<mat-card-content>
-<mat-list>
-  <mat-list-item *ngFor="let team of teams"><a routerLink="/team/{{team.id}}">{{team.displayName}}</a></mat-list-item>
-</mat-list>
-<button [disabled]="addTeamDisabled" mat-raised-button (click)="addTeam()" color="primary">Add Team</button>
-</mat-card-content>
-</mat-card>
+  <mat-card>
+    <mat-card-header>
+      <mat-card-title>Teams</mat-card-title>
+    </mat-card-header>
+    <mat-card-content>
+      <mat-nav-list>
+        <a mat-list-item *ngFor="let team of teams" routerLink="/team/{{team.id}}">{{team.displayName}}</a>
+      </mat-nav-list>
+    </mat-card-content>
+    <mat-card-actions>
+      <button [disabled]="addTeamDisabled" mat-raised-button (click)="addTeam()" color="primary">Add Team</button>
```

Comment:
Maybe we should consider making this a `mat-button` - maybe with `color="primary"` - and capitalising the text ("ADD TEAM")?

That's the style used in [this Angular Material example](https://material.angular.io/components/card/examples) and somehow I think it looks better.

It's not a big deal though - it can wait for a later PR.

Reply:
great suggestion, i believe it would look great. we can keep it as the app's default and use it also in dialog actions

I will make changes and send